pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaPartitionLevelStreamConfig.java

```diff
@@ -31,13 +31,16 @@
  */
 public class KafkaPartitionLevelStreamConfig {
 
+  public static final String POPULATE_ROW_METADATA = "stream.kafka.populateRowMetadata";
```

Comment:
I think this should be documented somewhere, not sure where the best place is

Reply:
I've raised #8775 in relation this question

Reply:
Thanks a lot for the contribution!
We can document it here - https://docs.pinot.apache.org/configuration-reference/table. Also I think you just need to keep the property name as `populateRowMetadata`. The prefix is handled by pinot and can be changed in the future.

Reply:
Shall we move this static to KafkaStreamConfigProperties?

Reply:
thank you @xiangfu0 @KKcorps  for reviewing, would this be generic enough to move it to `StreamConfigProperties` if I renamed it `consumption.metadata.populate` or something similar? I've reviewed both Pulsar and Kinesis stream plugins and this would apply to these 2 as well. The I think the property can be actually documented on the page that @KKcorps has mentioned above if we do this

Reply:
I've pushed a commit with this change, can revert/modify in any case will squash before merging